components/sst_importer/src/errors.rs

```diff
@@ -26,7 +26,7 @@ pub fn error_inc(type_: &str, err: &Error) {
         Error::ParseIntError(..) => "parse_int",
         Error::FileExists(..) => "file_exists",
         Error::FileCorrupted(..) => "file_corrupt",
-        Error::InvalidSSTPath(..) => "invalid_sst",
+        Error::InvalidSstPath(..) => "invalid_sst",
```

Comment:
/cc @5kbpers Will it change the error code?

Reply:
https://github.com/tikv/tikv/blob/8b509a7415e137b3719b1a947d11dfbf20e3bbb5/components/error_code/src/sst_importer.rs#L14 Need to modify here and run `make error-code`